kite_feeds.json

```diff
@@ -3071,5 +3071,99 @@
       "https://news.microsoft.com/feed/",
       "https://www.microsoft.com/releasecommunications/api/v2/m365/rss/"
     ]
+  },
+  "Privacy": {
+    "category_type": "topic",
+    "source_language": "en",
+    "display_names": {
+      "en": "Privacy",
+      "de": "Datenschutz",
+      "fr": "Vie privée",
+      "es": "Privacidad",
+      "pt": "Privacidade",
+      "it": "Privacy",
+      "nl": "Privacy",
+      "zh-Hans": "隐私",
+      "zh-Hant": "隱私",
+      "ja": "プライバシー",
+      "hi": "गोपनीयता",
+      "uk": "Приватність"
+    },
+    "feeds": [
+      "https://www.eff.org/rss/updates.xml",
+      "https://citizenlab.ca/feed/",
+      "https://iapp.org/rss/daily-dashboard",
+      "https://krebsonsecurity.com/feed/",
+      "https://www.schneier.com/feed/atom/",
+      "https://arstechnica.com/information-technology/category/security/feed/",
+      "https://www.wired.com/feed/category/security/latest/rss",
+      "https://blog.thenewoil.org/feed/",
+      "https://theprivacydad.com/feed/",
+      "https://inteltechniques.com/blog/feed/",
+      "https://privacyinternational.org/rss.xml",
+      "https://teachprivacy.com/feed",
+      "https://pogowasright.org/feed",
+      "https://dataprotectionreport.com/feed",
+      "https://bytebacklaw.com/feed",
+      "https://eyeonprivacy.com/feed",
+      "https://proton.me/blog/feed",
```

Comment:
I'm subscribed to their RSS feed, but mostly just because I'm too lazy to remove it.
They pretty much never have relevant coverage, it's mostly just ads.

I'd vote to drop them.